### PR TITLE
Added example involving complex PDE

### PR DESCRIPTION
**Pull request by FMagnani**
I think that many people would be glad to see an example involving a complex PDE as the Schroedinger equation. There are also some Issues addressing this point. I hope it will be useful.

### CODE REVIEW

**Comment by lululxvi** [comment]
Great. Could you format your code using [black](https://github.com/psf/black)?

**Comment by FMagnani** [comment]
1. Code formatted with Black ☑
2. File renamed ☑

File: examples/Schrodinger_Colab.ipynb [hunk 1]
```diff
@@ -0,0 +1,620 @@
...
+    ")\n",
+    "\n",
+    "# Initial conditions\n",
+    "def init_cond_u(x):\n",
+    "    \"2 sech(x)\"\n",
+    "    return 2 / np.cosh(x[:, 0:1])\n",
+    "\n",
+    "\n",
+    "def init_cond_v(x):\n",
+    "    return 0 * x[:, 0:1]\n",
```
Comment (lululxvi) [inline examples/Schrodinger_Colab.ipynb:298]: return 0

File: examples/Schrodinger_Colab.ipynb [hunk 2]
```diff
@@ -0,0 +1,620 @@
...
+   "source": [
+    "dde.optimizers.config.set_LBFGS_options(\n",
+    "    maxcor=50,\n",
+    "    ftol=1.0 * np.finfo(float).eps,\n",
+    "    gtol=1e-08,\n",
+    "    maxiter=10000,\n",
+    "    maxfun=10000,\n",
+    "    maxls=50,\n",
+    ")\n",
+    "model.compile(\"L-BFGS-B\", \"MSE\")\n",
```
Comment (lululxvi) [inline examples/Schrodinger_Colab.ipynb:458]: model.compile("L-BFGS")

File: examples/Schrodinger_Colab.ipynb [hunk 3]
```diff
@@ -0,0 +1,620 @@
...
+    "    h.T,\n",
+    "    interpolation=\"nearest\",\n",
+    "    cmap=\"viridis\",\n",
+    "    extent=[t_lower, t_upper, x_lower, x_upper],\n",
+    "    origin=\"lower\",\n",
+    "    aspect=\"auto\",\n",
+    ")\n",
+    "\n",
+    "plt.show()"
+   ],
```
Comment (lululxvi) [inline examples/Schrodinger_Colab.ipynb:583]: Could you add a note here that where is the expected accuracy of the PINN solution?

Comment (FMagnani) [reply examples/Schrodinger_Colab.ipynb:583]: Hello Lulu, I can employ the test data and compute the prediction error. I can also attach the plot of the absolute value of the error for each point of the domain. Is this that you mean?
